package.json

```diff
@@ -67,8 +67,6 @@
   },
   "license": "MIT",
   "resolutions": {
-    "@babel/core": "^7.12.10",
-    "@babel/runtime": "^7.13.10",
```

Comment:
I think its a bad idea to push these here. Only we are using the main package.json and this can mask issues like this.